tests/phpunit/CRM/Contribute/Form/Task/InvoiceTest.php

```diff
@@ -162,4 +162,43 @@ public function testInvoiceForLineItems() {
 
   }
 
+  /**
+   * Test invoices if payment is made with different currency.
+   *
+   * https://lab.civicrm.org/dev/core/issues/2269
+   *
+   * @throws \CRM_Core_Exception
+   */
+  public function testInvoiceWithDifferentCurrency() {
```

Comment:
I would rename this to : 

testThatInvoiceShowTheActuallContributionCurrencyInsteadOfTheDefaultOne

to make it more clear what this test is about.